src/wp-includes/blocks.php

```diff
@@ -1132,6 +1132,7 @@ function traverse_and_serialize_block( $block, $pre_callback = null, $post_callb
  */
 function traverse_and_serialize_blocks( $blocks, $pre_callback = null, $post_callback = null ) {
 	$result = '';
+	$parent = null; // At the top level, there is no parent block to pass to the callbacks; yet the callbacks expect a reference.
```

Comment:
Is there a reason not to name the same as the callback's parameter name? Keeping the naming same, i.e. `$parent_block`, can help with readability and understanding.

Reply:
Addressed in https://github.com/WordPress/wordpress-develop/pull/5598/commits/7aa6c9cd0c99f3281995458fe54effd3d7ccf651.